File: src/main/java/com/willmadison/legacycodekatas/fulfillment/orders/OrderItem.java
package com.willmadison.legacycodekatas.fulfillment.orders;

public class OrderItem {

    public enum Status {
        WIP,
        STRAGGLED,
        PICKED,
        DELETED,
        PLACED
    }

    public String id;

    public Status status;

    public boolean shipped;

    public boolean released;

    public int numStraggles;

}

File: src/main/kotlin/com/willmadison/legacycodekatas/fulfillment/orders/OrderItem.kt
package com.willmadison.legacycodekatas.fulfillment.orders

class OrderItem(val id: String, var status: Status, val shipped: Boolean, val released: Boolean) {
    var numStraggles: Int = 0

    enum class Status {
        WIP,
        STRAGGLED,
        PICKED,
        DELETED,
        PLACED
    }

}
